https://github.com/pauldreik/rdfind/pull/115

From 823a4deb61f6f9b91b0cfc4a7e7b20922c635777 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Thu, 1 Sep 2022 13:13:50 -0700
Subject: [PATCH] configure: Fix check for AC_CHECK_LIB

Check for nettle_pbkdf2_hmac_sha256 from libnettle instead of main()
which is not in nettle library

Signed-off-by: Khem Raj <raj.khem@gmail.com>
--- a/configure.ac
+++ b/configure.ac
@@ -46,7 +46,7 @@ AC_CHECK_HEADER(nettle/sha.h,,[AC_MSG_ERROR([
  On Debian-ish systems, use "apt-get install nettle-dev" to get a system
  wide nettle install.
 ])]) 
-AC_CHECK_LIB(nettle,main,,[AC_MSG_ERROR([
+AC_CHECK_LIB(nettle,nettle_pbkdf2_hmac_sha256,,[AC_MSG_ERROR([
  Could not link to libnettle. Please install nettle
  first. If you have already done so; please run ldconfig
  as root or check whether the path libnettle was installed

